src/ui/components/HeroSection/index.js

```diff
@@ -34,7 +44,7 @@ export default class HeroSection extends React.Component<Props> {
           <Link
             className="HeroSection-link-wrapper"
             onClick={this.props.onClick}
-            to={addQueryParams(linkTo, { src: INSTALL_SOURCE_HERO_PROMO })}
+            {...createToLink(linkTo, linkSource)}
```

Comment:
It would make sense to configure this link in the parent component then. Instead of introducing a new `linkSource` prop, you can call `addQueryParams()` in `src/amo/components/HomeHeroBanner/index.js`. This component will become more generic that way.

Reply:
yep that's a lot better!